modules/social_features/social_profile/social_profile.module

```diff
@@ -994,3 +1030,14 @@ function social_profile_profile_presave(EntityInterface $profile) {
     $profile->set('profile_name', $profile_name);
   }
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function social_profile_form_alter(array &$form, FormStateInterface $form_state, string $form_id): void {
+  if (in_array($form_id, [
+    'profile_profile_edit_form',
+  ])) {
+    $form['#attached']['library'][] = 'social_profile/admin';
+  }
+}
```

Comment:
Why do we need JavaScript to do this? Can't we not do it without JavaScript? Is the default not good enough?

Reply:
I've removed js library because I can not find the current field anymore